influxql/iterator.gen.go

```diff
@@ -418,6 +418,7 @@ type floatFillIterator struct {
 	curTime    int64
 	startTime  int64
 	endTime    int64
+	auxFields  []interface{}
```

Comment:
This is mostly there as the fill iterator needs to send nil values for the auxiliary fields and I don't think it's good to allocate a new array with the same values for every point, so I store the nil auxiliary values for the series here so the same ones can be sent with every point that gets created.